reports/sklearn_estimators_sample_weight_audit_report.ipynb

```diff
@@ -928,11 +1222,11 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "✅ 19 passed the deterministic test\n",
-      "❌ 4 failed the deterministic test\n",
-      "✅ 19 passed the statistical test\n",
-      "❌ 17 failed the statistical test\n",
-      "❌ 3 other errors\n",
+      "✅ 0 passed the deterministic test\n",
+      "❌ 0 failed the deterministic test\n",
```

Comment:
This part of the report should also be removed, since we no longer run the deterministic test in this notebook.